app/models/manageiq/consumption/showback_charge.rb

```diff
@@ -10,6 +10,7 @@ class ManageIQ::Consumption::ShowbackCharge < ApplicationRecord
   validates :showback_event, :presence => true, :allow_nil => false
 
   serialize :cost, JSON # Implement cost column as a JSON
```

Comment:
Why do you serialize the cost? What would be the result of this. I believe this is a mistake on my side.

As you can see in the class, cost is cost_subunits and cost_currency, so it can be monetized. Adding a serialization wouldn't help (unless we want to store a list of costs, in which case I don't know how to store money in a JSON)

Reply:
I didn't change cost